contrib/scalar/scalar.c

```diff
@@ -1172,6 +1210,27 @@ static int cmd_cache_server(int argc, const char **argv)
 	return !!res;
 }
 
+static int cmd_delete(int argc, const char **argv)
+{
+	struct option options[] = {
+		OPT_END(),
+	};
+	const char * const usage[] = {
+		N_("scalar delete <enlistment>"),
+		NULL
+	};
+
+	argc = parse_options(argc, argv, NULL, options,
+			     usage, 0);
+
+	if (argc != 1)
+		usage_msg_opt(N_("need an enlistment path"), usage, options);
```

Comment:
I think this would be more typical to just say `usage_with_options(usage, options);`

```suggestion
	if (argc != 1)
		usage_with_options(usage, options);
```

Reply:
Again, copied this from `cmd_clone`. Which is the right solution here?

Reply:
When in doubt, use the option that doesn't add a string to the translator's workload.